test/SynchroFeed.Command.ApplicationIs64bit.Test/SynchroFeed.Command.ApplicationIs64bit.Test.csproj

```diff
@@ -43,6 +43,10 @@
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0" />
     <PackageReference Include="Moq" Version="4.10.1" />
     <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
```

Comment:
VS wouldn't run these until I added this package, as the VSIX package is supposedly deprecated.  Point me in the right direction if I'm missing something.